src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContextHelpers.cs

```diff
@@ -70,6 +70,14 @@ internal static void VerifyDiagnosticLocationsInCompilation(Diagnostic diagnosti
                 }
             }
         }
+        
+        internal static void VerifyIOperationFeatureFlag(bool isIOperationFeatureEnabled)
```

Comment:
Yes, the feature flag related code was from dev14 updates, which is "removed" from master when we replaced it with future branch (dev15 never had the feature flag for IOperation).